test/images/agnhost/openidmetadata/openidmetadata.go

```diff
@@ -80,6 +84,24 @@ func main(cmd *cobra.Command, args []string) {
 	log.Printf("OK: got issuer %s", unsafeClaims.Issuer)
 	log.Printf("Full, not-validated claims: \n%#v", unsafeClaims)
 
+	if runtime.GOOS == "windows" {
```

Comment:
Could we leave a comment here for future us?

Reply:
I also think we can move the resolving logic to a function like "ensureWindowsDnsAvaliable" to hide some of this